index.html

```diff
@@ -21,10 +21,10 @@
         <tbody>
             <!--           first person -->
```

Comment:
Hi @MattOmolewa  you did the right thing, however, because another PR was merged into the master branch while your's was waiting you would need to run git fetch to have the latest version of the file on your machine.

After doing this, you should review if there's any. conflict in the file, fix them and resubmit your PR.